tests/services/analysis_starter/file_creators/balsamic_config_file_creator/conftest.py

```diff
@@ -0,0 +1,255 @@
+import pytest
+
+from cg.models.cg_config import BalsamicConfig
+
+
+@pytest.fixture
+def expected_tgs_normal_only_command(cg_balsamic_config: BalsamicConfig) -> str:
+    return (
+        f"{cg_balsamic_config.conda_binary} "
+        f"run --name {cg_balsamic_config.conda_env} "
+        f"{cg_balsamic_config.binary_path} config case "
+        f"--analysis-dir {cg_balsamic_config.root} "
+        f"--analysis-workflow balsamic "
+        f"--artefact-snv-observations {cg_balsamic_config.loqusdb_artefact_snv} "
+        f"--balsamic-cache {cg_balsamic_config.balsamic_cache} "
+        f"--cadd-annotations {cg_balsamic_config.cadd_path} "
+        f"--cancer-germline-snv-observations {cg_balsamic_config.loqusdb_cancer_germline_snv} "
+        f"--cancer-somatic-snv-observations {cg_balsamic_config.loqusdb_cancer_somatic_snv} "
+        f"--cancer-somatic-sv-observations {cg_balsamic_config.loqusdb_cancer_somatic_sv} "
+        f"--case-id case_1 "
+        f"--clinical-snv-observations {cg_balsamic_config.loqusdb_clinical_snv} "
+        f"--clinical-sv-observations {cg_balsamic_config.loqusdb_clinical_sv} "
+        f"--fastq-path {cg_balsamic_config.root}/case_1/fastq "
+        f"--gender female "
```

Comment:
@mogmi made some changes to the sex of these cases in the dev branch, I think you don't have those changes in this branch